packages/node_modules/@webex/widget-space/src/enhancers/setup.js

```diff
@@ -79,17 +79,30 @@ export function setup(props, prevProps = {}) {
           sparkInstance,
           destinationId: destination.id,
           destinationType: destination.type,
-          intl: props.intl
+          intl: props.intl,
+          ...destination.cluster && {destinationCluster: destination.cluster}
         });
         // Fetch 1:1 User details
-        if (destination.type === destinationTypes.EMAIL && !users.getIn(['byEmail', destination.id])) {
+        if (
+          destination.type === destinationTypes.EMAIL &&
+          !users.getIn(['byEmail', destination.id])
+        ) {
           props.getUser({email: destination.id}, sparkInstance);
         }
-        if (destination.type === destinationTypes.USERID && !users.getIn(['byId', destination.id])) {
+        if (
+          destination.type === destinationTypes.USERID &&
+          !users.getIn(['byId', destination.id])
+        ) {
           props.getUser({id: destination.id}, sparkInstance);
         }
-        if (destination.type === destinationTypes.SPACEID && !spaces.getIn(['byId', validateAndDecodeId(destination.id).id])) {
-          props.fetchSpace(sparkInstance, validateAndDecodeId(destination.id));
+        if (
+          destination.type === destinationTypes.SPACEID &&
+          !spaces.getIn(['byId', validateAndDecodeId(destination.id).id])
+        ) {
+          props.fetchSpace(
+            sparkInstance,
+            validateAndDecodeId(destination.id, destination.cluster)
```

Comment:
This function doesn't take a second param.

Reply:
This list of if blocks was prettier'd so i could read it
the second param was already there, but i can remove it

https://github.com/webex/react-widgets/blob/master/packages/node_modules/@webex/widget-space/src/enhancers/setup.js#L92

Reply:
I'm saying `validateAndDecodeId` doesn't take two params.

https://github.com/webex/react-widgets/blob/1310dae30e7a86f65b4a1b59e2e8d8a0a691f401/packages/node_modules/%40webex/react-component-utils/src/uuid-utils.js#L75